junit-jupiter-api/src/main/java/org/junit/jupiter/api/extension/ExtensionContext.java

```diff
@@ -160,7 +161,9 @@
 	 * @return an {@code Optional} containing the test instance; never
 	 * {@code null} but potentially empty
 	 * @see #getRequiredTestInstance()
+	 * @see #getTestInstances()
 	 */
+	@API(status = MAINTAINED, since = "5.4")
```

Comment:
I think this has been released prior to 5.4

Reply:
indeed 😉

Reply:
It was marked **STABLE** in 5.0 and can't be reduced to **MAINTAINED** in 5.4.

Reply:
I agree with @sormuras: changing from `STABLE` to anything else within the 5.x timeline is a violation of the API rules.

Reply:
Agreed.